server/views/partials/layout.njk

```diff
@@ -56,6 +60,12 @@
   </div>
 {% endblock %}
 
+{% block footer %}
+  {{ govukFooter({
+    classes: "govuk-!-display-none-print"
```

Comment:
I've included the footer in our template instead of pulling it from the GOVUK shared template, this allows us to add custom classes and it will future proof us when we need to use the new footer from the DPS team.